skiko/src/jsMain/kotlin/org/jetbrains/skiko/wasm/Wrapper.kt

```diff
@@ -2,12 +2,79 @@ package org.jetbrains.skiko.wasm
 
 import kotlinx.dom.createElement
 import kotlinx.browser.document
+import org.jetbrains.skia.impl.NativePointer
+import org.khronos.webgl.WebGLRenderingContext
 import org.w3c.dom.HTMLCanvasElement
+import org.w3c.dom.RenderingContext
 import kotlin.js.Promise
 
 external val wasmSetup: Promise<Boolean>
 external fun onWasmReady(onReady: () -> Unit)
 
+external interface GLInterface {
+    fun createContext(context: HTMLCanvasElement, contextAttributes: dynamic): NativePointer;
+    fun makeContextCurrent(contextPointer: NativePointer): Boolean;
+}
+
+external object GL: GLInterface {
+    override fun createContext(context: HTMLCanvasElement, contextAttributes: dynamic): Int = definedExternally
+    override fun makeContextCurrent(contextPointer: NativePointer): Boolean = definedExternally
+}
+
+external interface ContextAttributes {
+    val alpha: Int?
+    val depth: Int?
+    val stencil: Int?
+    val antialias: Int?
+    val premultipliedAlpha: Int?
+    val preserveDrawingBuffer: Int?
+    val preferLowPowerToHighPerformance: Int?
+    val failIfMajorPerformanceCaveat: Int?
+    val enableExtensionsByDefault: Int?
+    val explicitSwapControl: Int?
+    val renderViaOffscreenBackBuffer: Int?
+    val majorVersion: Int?
+}
+
+fun ContextAttributes.asJsObject(): dynamic {
+    val jsObject = js("{}")
```

Comment:
Wouldn't 'val jsObject: dynamic = Any()' be better?

Reply:
any and dynamic are different beasts in Kotlin/JS - Any is a standard "regular" type and still lacks possibility to refer to arbitrary fields (like dynamic does)

Reply:
OK, just looks somewhat unusual.